tests/fault_tolerance/hardware/fault-injection-service/examples/test_partition_worker_to_frontend.py

```
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)